### DISCUSSION TITLE
独立“下载器”功能/模块

### DISCUSSION THREAD

**Discussion by Ringeal007** [Meloong-Git/PCL#1067] [提问] [answered]
隐藏“启动”页面，只显示“下载自定义文件”功能。

_（或者干脆发布一个“Plain Craft Downloader”（PCD）[doge]）_

**Comment by WorldHim** [comment, marked as answer]
你可以在 `设置` - `个性化` - `功能隐藏` 中关闭（即勾选）除 `百宝箱` 外的所有功能。

**Comment by WForst-Breeze** [comment]
同上，虽然功能隐藏无法隐藏“启动”页面，但……PCL2 的本职工作是启动器（）自定义下载什么的顶多算副业qwq 所以不太可能做，抱歉啦
